crypto/rand/drbg_lib.c

```diff
@@ -315,7 +387,7 @@ int rand_drbg_restart(RAND_DRBG *drbg,
 
     /* repair uninitialized state */
     if (drbg->state == DRBG_UNINITIALISED) {
-        drbg_setup(drbg, NULL);
+        drbg_setup(drbg, NULL, NULL);
```

Comment:
Should that be drbg->parent?

Reply:
Oh, it's using code that doesn't actually update the parent when called with NULL ...